src/components/Editor/Invitees/InviteesList.vue

```diff
@@ -115,6 +119,9 @@ export default {
 		noInviteesMessage() {
 			return this.$t('calendar', 'No attendees yet')
 		},
+		noOwnerMessage() {
+			return this.$t('calendar', 'You dont own this calendar, so you cannot invite other people')
```

Comment:
**suggestion:** 
```suggestion
			return this.$t('calendar', "You don't own this calendar, so you cannot add attendees to this event")
```
As attendees may not just be people in the future.

Reply:
We should probably also disable rooms and resources